packages/gafl-webapp-service/src/pages/contact/contact/__tests__/route.spec.js

```diff
@@ -182,14 +184,20 @@ describe('name > route', () => {
     })
 
     it.each([
-      [true, '12M'],
-      [false, '8D'],
-      [false, '1D']
-    ])('result.twelveMonthLicence returns %s when licence length is %s', async (expected, licenceLength) => {
-      const licensee = { birthDate: 'birthDate' }
-      const result = await getData(getMockRequest({ licensee, licenceLength }))
-      expect(result.twelveMonthLicence).toBe(expected)
-    })
+      [true, '12M', false],
+      [false, '8D', false],
+      [false, '1D', false],
+      [true, '12M', false],
+      [false, undefined, true]
```

Comment:
This _might_ benefit from a template literal, less clear-cut though, I think it's still about legible. I'll leave the call up to you.